### PR TITLE
fixing admin interface for handling teachers model

### PR DESCRIPTION
**Pull request by twalen**
Fixing issue #282

### CODE REVIEW

File: oioioi/teachers/forms.py [hunk 1]
```diff
@@ -48,3 +48,39 @@ def __init__(self, *args, **kwargs):
...
+
+    def __init__(self, *args, **kwargs):
+        super(AdminTeacherForm, self).__init__(*args, **kwargs)
+        self.fields['user'].hints_url = reverse('user_search')
+        instance_user = None
+        if 'instance' in kwargs:
+            instance = kwargs['instance']
+            if hasattr(instance, 'user'):
+                instance_user = instance.user
+                print(f"user={repr(instance_user)}")
```
Comment (twalen) [inline oioioi/teachers/forms.py:68]: Please remove print

**Comment by twalen** [comment]
Missing tests for AdminTeacherForm

File: oioioi/teachers/forms.py (hunk 1 above)
Comment (twalen) [reply oioioi/teachers/forms.py]: done

**Comment by twalen** [comment]
> Missing tests for AdminTeacherForm

added

**Review by franekwitt — APPROVED** [review]
Ok

**Pull request merged**